packages/inter-protocol/src/proposals/demoIssuers.js

```diff
@@ -296,8 +302,8 @@ export const connectFaucet = async ({
 
         /** @type {UserPaymentRecord[]} */
         let toFaucet = [];
-        // Use the bank layer for BLD, RUN.
-        if (['BLD', 'RUN'].includes(issuerName)) {
+        // Use the bank layer for BLD, IST.
+        if (issuerName === Stake.symbol || issuerName === Stable.symbol) {
```

Comment:
The old format makes it easier to see that the two tests are parallel.    Just a suggestion

Reply:
Agreed.